cli/cmd/install.go

```diff
@@ -146,6 +138,10 @@ func arePodsReady(ctx context.Context, client *kube.Client, ns string) func() (b
 
 func createNamespace(ctx context.Context, cmd *cobra.Command, client *kube.Client, ns string) error {
 	_, err := client.CoreV1().Namespaces().Create(ctx, resources.NewNamespace(ns), metav1.CreateOptions{})
+	if apierrors.IsAlreadyExists(err) {
```

Comment:
should we check that the namespace contains the `odigos.io/system-object` label?
If not, it will be broken downstream